pandas/io/excel/_base.py

```diff
@@ -356,7 +356,7 @@
 1       NaN      2
 2  #Comment      3
 
-Comment lines in the excel input file can be skipped using the `comment` kwarg
+Comment lines in the excel input file can be skipped using the ``comment`` ``kwarg``
```

Comment:
kwarg here should not be included. Can you write out the full phrase here: keyword argument.

Reply:
Good call.  Should ``comment`` still be in a code block in this case since it's referencing the name of a parameter?

Reply:
Yes - I think so.